web/.storybook/public/mockServiceWorker.js

```diff
@@ -8,7 +8,7 @@
  * - Please do NOT serve this file on production.
  */
 
-const PACKAGE_VERSION = '2.3.4'
+const PACKAGE_VERSION = '2.3.5'
```

Comment:
Assuming that we need the `mockServiceWorker.js` only within Storybook, I think we should be able to extend `web/scripts/run-storybook.sh` to execute `msw init web/.storybook/public --save` before each start of the Storybook.

The execution of this command takes a fraction of the second. In return, we'd no longer have to remember about updating this file. See the second paragraph of [Committing the worker script](https://mswjs.io/docs/best-practices/managing-the-worker#committing-the-worker-script).

Reply:
I just went ahead and did it, let me know if you see a problem with that.

`mockServiceWorker.js` is not needed for `pnpm storybook-smoke-test`, as no actual stories are executed by this command.